src/main/java/no/nav/familie/integrasjoner/sftp/FiloverføringAdraMatchClient.kt

```diff
@@ -0,0 +1,37 @@
+package no.nav.familie.integrasjoner.sftp
+
+import com.jcraft.jsch.ChannelSftp
+import com.jcraft.jsch.JSch
+import com.jcraft.jsch.Session
+import no.nav.familie.integrasjoner.config.FiloverføringAdraMatchConfig
+import no.nav.familie.kontrakter.felles.Fil
+import org.springframework.stereotype.Service
+import java.io.ByteArrayInputStream
+
+@Service
+class FiloverføringAdraMatchClient(private val config: FiloverføringAdraMatchConfig) {
```

Comment:
Hva er adramatch? Kan vi ikke gi den ett navn som gir litt bedre mening for våret team? Eller kommentere

Reply:
Hvis noen kommer inn og ser denne koden er det umulig å skjønne hva den brukes til

Reply:
Den overfører av filer til Adra Match. Det er en eller annen for for avstemming av tilbakekreving. Hvordan filene brukes når de kommer frem, er jo ikke så relevant for en sftp-overføring. Sånn sett så var jo kankje det gamle navnet bedre.